providers/os/resources/packages/packages.go

```diff
@@ -109,6 +109,7 @@ func ResolveSystemPkgManager(conn shared.Connection) (OperatingSystemPkgManager,
 		pm = &AlpinePkgManager{conn: conn, platform: asset.Platform}
 	case asset.Platform.Name == "macos": // mac os family
 		pm = &MacOSPkgManager{conn: conn}
+		// TODO: add a new manager here for fs conn that lists registry directly
```

Comment:
when will we do this?

Reply:
i will remove this TODO, this isnt the way i was planning on fixing that either way, its a leftover